### PR TITLE
Add progress window while job is running

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit adds progress window while job is running.

ONE-vscode-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

**Comment by jyoungyun** [comment]
This PR will conflict with #634. After merged #634, I will change it to Normal PR.

File: src/Project/JobRunner.ts [hunk 1]
```diff
@@ -100,6 +107,24 @@ export class JobRunner extends EventEmitter {
       Balloon.error('ONE compile in progress');
       return;
     }
+
+    vscode.window.withProgress(
+        {location: vscode.ProgressLocation.Notification, title: 'Running Job', cancellable: false},
```
[the remark this answers is not in the document]
Comment (jyoungyun) [reply src/Project/JobRunner.ts:112]:
```suggestion
        {location: vscode.ProgressLocation.Notification, title: 'Running...', cancellable: false},
```

File: src/Project/JobRunner.ts [hunk 2]
```diff
@@ -100,6 +107,24 @@ export class JobRunner extends EventEmitter {
...
+          return new Promise<void>(resolve => {
+            this.progress = progress;
+            this.progressTimer = setInterval(() => {
+              if (!this.running) {
+                if (this.progressTimer) {
+                  clearInterval(this.progressTimer);
+                  resolve(undefined);
+                }
+              }
+            }, 1000);
```
[the remark this answers is not in the document]
Comment (jyoungyun) [reply src/Project/JobRunner.ts:124]: This value is delay parameter of the `setInterval()` functions. Every 1000ms calls the first parameter function.

Please refer the sites below:
https://www.educba.com/setinterval-typescript/
https://github.com/microsoft/TypeScript/blob/cec2fda9a53620dc545a2c4d7b0156446ab145b4/lib/lib.webworker.d.ts#L5897

**Review by hyunsik-yoon — APPROVED** [review]

**Comment by jyoungyun** [comment]
PTAL @Samsung/one-vscode

**Review by llFreetimell — APPROVED** [review]

**Pull request merged**